app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -418,7 +419,12 @@ const CourseCreator = createReactClass({
         <label htmlFor="checkbox_id">{I18n.t('courses.creator.copy_courses_with_assignments')}</label>
       </span>
     );
-
+    const { ifadmin: ifAdminStr, wiki_ed: wikiEdStr } = document.getElementById('nav_root').dataset;
+    let isAdminOrInstructor;
+    if (ifAdminStr === 'true' || Features.wikiEd === 'true') {
```

Comment:
I don't think I understand what's going on here.

Reply:
@rageross This checks if the, user is an admin or a  wiki_ed staff

Reply:
@rageross This checks if the, user is an admin or a  wiki_ed staff

Reply:
`Features.wikiEd` is for whether the server is in `wiki_education` mode or not. It is unrelated to whether the current user is wiki education staff.